packages/graphql/lib/schema-builder/collections/array.collection.ts

```diff
@@ -0,0 +1,15 @@
+export class ArrayCollection<T> extends Array<T> {
```

Comment:
Extending built-in collections is typically considered a bad practice. Any reason we cant use Array directly?

Reply:
Pushed another fix, now I'm wrapping the array instead of extending it.  Good comment!